lib/hello_web/plugs/locale.ex

```diff
@@ -0,0 +1,15 @@
+defmodule HelloWeb.Plugs.Locale do
```

Comment:
@YanethM what is the benefit of using plugs?

Reply:
As I understand it,  the plugs are used during the life cycle of HTTP actions. In this case, the use of plug is at the module level for the regional configuration of the language, but it can also be used in the router for some routes like match or dispatch.